Add Intellifire Gas Fireplace integration (#21092)
home-assistant/home-assistant.github.io: source/_integrations/intellifire.markdown

---
title: IntelliFire
description: Instructions on the IntelliFire Fireplace integration for Home Assistant.
ha_category:
  - Binary Sensor
ha_iot_class: Local Polling

> ha_release: 0.47

Reviewer: ```suggestion
ha_release: 2022.2
```

---

> title: Intellifire 

Reviewer: ```suggestion
title: IntelliFire
```

> description: Instructions on the Intellifire Fireplace integration for Home Assistant.

Reviewer: ```suggestion
description: Instructions on the IntelliFire Fireplace integration for Home Assistant.
```

ha_category:
  - Binary Sensor
ha_iot_class: Local Polling
ha_release: 0.47
ha_codeowners:
  - '@jeeftor'
ha_domain: intellifire
ha_config_flow: true
---

> 
> ### Overview
> 

Reviewer: ```suggestion
```

> Intellifire Wifi fireplace modules provide app-based and Alexa control to various fireplace. The modules do expose an unencrypted HTTP endpoint on the network that provides status information. This integration will read that URL and create a set of sensors displaying the current fireplace state.

Reviewer: ```suggestion
IntelliFire Wi-Fi fireplace modules provide app-based and Alexa control to various fireplaces. The modules do expose an unencrypted HTTP endpoint on the network that provides status information. This integration will read that URL and create a set of sensors displaying the current fireplace state.
```

I'm not sure if we need to explain the unencrypted connection and the URL read and such. Is it really useful for an end-user user to tell about how it works internally?

> Configuration is done via the GUI.
> 

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

### Sensor Types

The following sensors are available as either a **Binary Sensor** when dealing with on/off.

### Binary Sensors

- `pilot_light`: Pilot light status.
- `fan_on`: Whether the fan is on/off.
- `timer_on`: Whether timer-mode is engaged.

> - `thermostat_on`: Whether the thermostat is turned on.

Reviewer: What is with the code fenced items?

Author: I was trying to call out the different sensors it would display .. .and so I used the code formatting on the sensor names. If that's not aligned with other documentation I'm happy to remove the fencing...

(I personally though it looked clearer ...thats all)

Reviewer: Well it assume/pretends its code (the underscores add to that as well). I guess it is referring to entities object IDs or something. However, those aren't guaranteed either.

It is better to have just a normal list of human-readable sensor names and their descriptions.

Author: Sounds good -> I've updated it to human readable mode (and found a minor bug which I resolved)
